src/js/providers/video-actions-mixin.js

```diff
@@ -30,7 +30,9 @@ const VideoActionsMixin = {
                 objectFit = 'fill';
             }
             style(this.video, {
-                objectFit: objectFit
+                objectFit: objectFit,
```

Comment:
You can shorthand this

Reply:
true. It seems weird to mix shorthand and not...

Reply:
thanks for updating it.